app/forms/developer_search_form.rb

```diff
@@ -10,11 +10,20 @@ class DeveloperSearchForm
 
   def search
     query = Developer.all
-    query = query.joins(:developer_languages).where(developer_languages: { language_id: language_id }) if language_id.present?
+    if language_id.present?
+      query = query.includes(:developer_languages)
```

Comment:
It looks like there should be still inefficient queries.
Can you check the log when you do the search?

Reply:
I checked again. Because I call in view like this https://gyazo.com/c3a2dd21ba021f9951384021bcd6f2a9
I'm looking for a different solution

Reply:
Now, I fixed it. Thank you very much